lib/Doctrine/Common/Reflection/StaticReflectionParser.php

```diff
@@ -144,6 +144,7 @@ protected function parse()
         }
         $tokenParser = new TokenParser($contents);
         $docComment = '';
+        $last_token = FALSE;
```

Comment:
Just a reminder: lowercase `false` please ;-)

I'm changing it on merge anyway.